### ISSUE TITLE
Parse Migration: The job failed too many times and reached the max retry limit. 

### ISSUE THREAD

**Issue by SeloSlav** [felixrieseberg/parse-server-azure#8]
This was described as potentially a hardware issue by the Parse Migration guide: http://www.parse.com/migration

My collections aren't massive by any means (1,000-5,000 objects per class with about 10 classes). I'm using DocumentDB and Microsoft Azure on F1 (Free, 1GB). My data and storage usage hover at around 0.2 GB on Parse. Should I upgrade to a higher pricing tier? Would this fix the problem?

**Comment by mamaso** [comment]
Hi @santafebound,

Thanks for reaching out.  This is a known issue with some migrations, a fix is expected to be deployed tomorrow EOD.  If the migration still fails at that point you should follow up.

**Comment by sidan5** [comment]
@mamaso I'm having the same issue. Using mLab dedicated server with 40GB (migrating 3 GB).

**Comment by adrianhall** [comment]
Hi @sidan5 If you are using the mLab dedicated server for your MongoDB service, then you are not having the same issue.  This is explicitly for using DocumentDB as the MongoDB service.

**Comment by Dragon160** [comment]
Hi @santafebound the problem is that the free documentDB tier only allows 10 standard collections. So, the parse migration tool tries to add a new collection but it is not possible. You can verify this, by adding manually (within the azure portal) a collection - it will not work.

**Comment by mariusmuntean** [comment]
@adrianhall how can I tell my managed Parse to use a MongoDB from mLab? Is it enough to specify the connection string in the Application Settings (as the value for the Key DATABASE_URI)?

**Comment by mamaso** [comment]
@mariusmuntean

Yes, you can change the DATABASE_URI appsetting to a connection string for whichever mongo provider you prefer.

**Comment by liambolling** [comment]
Anyone find a solution to this?

**Comment by Dragon160** [comment]
@liambolling the problem for the original issue was the limited free documentDB tier (only 10 collections allowed). Just switch to a other tier or a hosted mongo-db for e.g. mlab.
